helm-chart/banzai/values-dev.yaml

```diff
@@ -36,34 +36,38 @@ tolerations: []
 
 affinity: {}
 
-CALIBRATE_PROPOSAL_ID: calibrate
+calibrateProposalId: calibrate
```

Comment:
Why did we switch to camel case instead of env variables?

Reply:
See the description. These are values files, and camelcase is the standard.